### PR TITLE
Add simple test

### PR DESCRIPTION
**Pull request by joshbuddy**

### CODE REVIEW

**Comment by joshbuddy** [comment]
@phildini wanted to separate out the parsing with the fetching for testability

File: lib/patreon/api.rb [hunk 1]
```diff
@@ -10,28 +11,33 @@ def initialize(access_token)
...
     end
 
-    def fetch_page_of_pledges(campaign_id, page_size, cursor=nil, includes=nil, fields=nil)
-      url = "campaigns/#{campaign_id}/pledges"
-      url += "?page%5Bcount%5D=#{CGI::escape(page_size.to_s)}"
-      url += "&page%5Bcursor%5D=#{CGI::escape(cursor.to_s)}" if cursor
-      get_json(Patreon::Utils::JSONAPI::URLUtil.build_url(url,includes,fields))
+    def fetch_page_of_pledges(campaign_id, page_size, opts = {})
+      params = { "page[count]" => page_size.to_s }
+      params["page[cursor"] = opts[:cursor] if opts[:cursor]
```
Comment (phildini) [inline lib/patreon/api.rb]: Is this quoting and bracketing in the right order?

**Comment by phildini** [comment]
One nit, but overall these look good.

File: lib/patreon/api.rb (hunk 1 above)
Comment (joshbuddy) [reply lib/patreon/api.rb]: no, no it is not. don't trust any of this until we've got some tests going

**Pull request merged**